content/en/users/compute/orchestration/im/dashboard/_index.md

```diff
@@ -135,7 +135,11 @@ infrastructures:
 - Add nodes: The Add nodes action enables to add new VMs to the users'
   deployment. It will show the list of the different types of nodes currently
   deployed in the infrastructure and the user must set the number of nodes of
-  each type he wants to deploy.
+  each type he wants to deploy. It will also show the image selected to create
+  the VMs (if available). In the dropdown list the user can change the base
+  image, in case that you want to use a new one on the new deployed nodes, or
+  the old one has been removed and you need to select a new one to be able to
+  deploy them.
```

Comment:
There is a mix of using `he/the user` and `you`, it should be aligned, ideally to use the second person as mentioned on https://docs.egi.eu/about/contributing/style/#write-in-second-person

Reply:
As in the rest of the document third person is used I alined it to use it.

Reply:
OK, but let's aim at having the full document be moved to 2nd person at some point, as it's the expected way.